infra.bs

```diff
@@ -486,6 +486,24 @@ implementations of just <a>JavaScript strings</a> for performance and memory rea
 
 <hr>
 
+<p>To <dfn export for=string lt=concatenate|concatenation>concatenate</dfn> a <a for=/>list</a> of
```

Comment:
I envisioned this being located in the same hr-delimited section as the split operations.